libpod/oci_conmon_common.go

```diff
@@ -1328,6 +1328,9 @@ func (r *ConmonOCIRuntime) configureConmonEnv(runtimeDir string) []string {
 	if conf, ok := os.LookupEnv("CONTAINERS_CONF"); ok {
 		env = append(env, fmt.Sprintf("CONTAINERS_CONF=%s", conf))
 	}
+	if conf, ok := os.LookupEnv("CONTAINERS_CONF_OVERRIDE"); ok {
```

Comment:
@mheon do you recall why we're not passing all env variables to conmon?  I feel like we should do this since there is a growing number of those which I do not see being passed down.

Reply:
I think it dates back to before we forked off Conmon, so this is probably older than my involvement in the project. I don't see a good reason not to forward on full environment?

Reply:
Thanks!   I will open a follow-up PR :+1: